asgs_main.sh

```diff
@@ -1754,6 +1753,7 @@ writeJobResourceRequestProperties()
    logMessage "$THIS: Writing properties associated with compute job to $1/run.properties."
    # on queenbeeC, if a parallel job uses 48 or fewer cores, it
    # should be submitted to the single queue to avoid "low utilization" emails
+   CPUREQUEST=`expr $NCPU + $NUMWRITERS`
    if [[ $HPCENV = "qbc.loni.org" && $CPUREQUEST -le 48 ]]; then
```

Comment:
I think next time we're in this part of the code, turning this into a `case` statement would probably a good move (based on `$HPCENV`.